CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Table of Contents
-
+ - [2.0.0-ga](#200-ga---20210831)
+ - [2.0.0-beta.2](#200-beta2---tbd)
```

Comment:
```suggestion
```

These sections don't exist yet - why reference non-existing sections in the table of contents?

Reply:
changed https://github.com/Kong/kubernetes-ingress-controller/pull/1657/commits/3356126e89725421682a60d816b135bd1dd97708